custom_model_runner/requirements.txt

```diff
@@ -13,4 +13,4 @@ scipy>=1.1,<2
 strictyaml==1.0.6
 texttable
 py4j~=0.10.9.0
-pyarrow
+pyarrow==2.0.0
```

Comment:
is this _real_ enough for you, @4dahalibut ?

Reply:
<3